hc/front/views.py

```diff
@@ -60,6 +60,28 @@ def my_checks(request):
     return render(request, "front/my_checks.html", ctx)
 
 
+@login_required
+def failed_checks(request):
+    q = Check.objects.filter(user=request.team.user).order_by("created")
+    checks = list(q)
```

Comment:
Why convert `q` to a list?

Reply:
At first we did not realise that the Queryset object returned from the query was actually a list. So we were converting it to a list in order to loop over it.

We now understand it works like a list.

Reply:
Good, update it to iterate over the [queryset](https://docs.djangoproject.com/en/1.11/ref/models/querysets)